SampleViewer/SampleManager.h

```diff
@@ -178,6 +193,7 @@ class SampleManager : public QObject
   std::unique_ptr<QTemporaryDir> m_tempDir;
   bool m_cancelDownload = false;
   bool m_downloadFailed = false;
+  Esri::ArcGISRuntime::Authentication::ArcGISAuthenticationChallengeHandler* m_previousChallengeHandler = nullptr;
```

Comment:
Consider calling this `m_toolkitChallengeHandler` for clarity. "previous" implies we're doing something else.